wagtailio/core/models/pages.py

```diff
@@ -22,17 +25,68 @@ class HomePage(Page, SocialMediaMixin, CrossPageMixin):
         "packages.PackagesPage",
         "services.ServicesPage",
     ]
-    content_panels = Page.content_panels + [StreamFieldPanel("body")]
 
-    promote_panels = (
-        Page.promote_panels + SocialMediaMixin.panels + CrossPageMixin.panels
+    # ----------------- Hero -----------------
+    heading = models.TextField(verbose_name="Heading", blank=True)
+    sub_heading = models.TextField(verbose_name="Sub heading", blank=True)
+    intro = RichTextField(
+        verbose_name="Intro",
+        blank=True,
+        features=["bold", "italic", "link"],
+    )
+    icon = models.CharField(choices=SVGIcon.choices, max_length=255, blank=True)
+
+    code_snippet = models.TextField(
+        blank=True,
+        default="pip install wagtail",
+    )
+    call_to_action = StreamField(
+        [("cta", CTABlock())],
+        blank=True,
+        max_num=2,
+        help_text="Allows for a maximum of 2 CTA blocks",
+    )
+    video = models.ForeignKey(
+        "wagtailmedia.Media",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
     )
+    loop_video = models.BooleanField(default=False)
+    autoplay_video = models.BooleanField(default=False)
 
-    def get_context(self, request, *args, **kwargs):
-        context = super(HomePage, self).get_context(request, *args, **kwargs)
-        context.update({"blog_posts": BlogPage.objects.live().order_by("-date")})
+    body = StreamField(HomePageStoryBlock())
 
-        return context
+    hero_panels = [
+        MultiFieldPanel(
+            [
+                FieldPanel("heading"),
+                FieldPanel("sub_heading"),
+                FieldPanel("intro"),
+                FieldPanel("icon"),
+                FieldPanel("code_snippet"),
+                StreamFieldPanel("call_to_action"),
+                MediaChooserPanel("video", media_type="video"),
+                FieldPanel("loop_video"),
+                FieldPanel("autoplay_video"),
```

Comment:
this needs to be its own `MultiFieldPanel` with video first, then code_snippet and finally call to action. Label: "Promo" or "Promotional area"

Reply:
done in 34198484